.gitignore

```diff
@@ -1 +0,0 @@
-node_modules/
```

Comment:
Just saw your comment in the other PR:
> FYI, I will probably git rm.gitignoreonMASTER, since most folks configure Git to ignore those directories globally andgit add -f` in special cases.

I don't use that global config :)